lib/msf/ui/console/command_dispatcher/session.rb

```diff
@@ -136,23 +136,36 @@ def cmd_pry(*args)
           end
 
           def cmd_sessions_help
-            print_line('Usage: sessions <id>')
+            print_line('Usage: sessions <id> or sessions -i <id>')
             print_line
             print_line('Interact with a different session Id.')
-            print_line('This works the same as calling this from the MSF shell: sessions -i <session id>')
             print_line
           end
 
           def cmd_sessions(*args)
```

Comment:
Would it make sense to add in a Rex Parser here?:
```ruby
@@session_opts = Rex::Parser::Arguments.new(
            ['-h', '--help] => [ false, 'Show this message' ],
            ['-i', '--interact'] => [ true, 'Interact with a provided session ID' ]
          )

@@session_opts.parse(args) do |opt, _idx, val|
              case opt
              when '-h'
                ...
              when '-i'
                ...
              else
                 ...
              end
end
```

Reply:
Since there's only one option needed here, for simplicity's sake I feel we don't need this. If we use this, we'll also have to separately handle `sessions <id>` as done [here](https://github.com/rapid7/metasploit-framework/blob/15c56a870e2371be3e82c63b7d0fcc63c5a21a6d/lib/msf/ui/console/command_dispatcher/core.rb#L1475C5-L1477C25). If there's plans of adding new options here, it'd make sense to use the rex parser. I'll happily add the rex parser if you still feel we should use it.